nncf/torch/nncf_module_replacement.py

```diff
@@ -186,6 +187,9 @@ def replace_modules_by_nncf_modules(
             else:
                 replaced_module = nncf_module_from(module)
 
+            device = get_model_device(module)
+            replaced_module.to(device)
+
             inter_dict[replaced_module] = scope_set
```

Comment:
Better to get the model device before the replacer function, just in case the `nncf_module_from` does not return a copy of `module` but instead does in-place modifications.

Reply:
done